src/NuGet.Clients/NuGet.SolutionRestoreManager/Logger/NuGetFileLogger.cs

```diff
@@ -24,6 +24,8 @@ internal sealed class NuGetFileLogger : IDisposable
 
         public bool IsEnabled { get; }
 
+        public bool ShouldFormatWithTime { get; }
```

Comment:
Note that this logger is a testing utility that's *only* built under debug, so changes here don't affect the product.